helpers/update-site/documentation.js

```diff
@@ -405,6 +406,37 @@ const getMeta = async (file) => {
     return meta;
 };
 
+const getLocalizationFilePath = (file) => {
+    if (file.isMulti) {
+        return path.join(file.orig, '..', '..', 'src', '_locales', 'en', 'messages.json');
+    }
+
+    return path.join(file.orig, '..', 'src', '_locales', 'en', 'messages.json');
+};
+
+const getDescriptionFromLocalization = (file) => {
+    const messagesFilePath = getLocalizationFilePath(file);
+    const messages = require(messagesFilePath);
+
+    if (!file.isMulti) {
+        return messages.description.message;
+    }
+
+    const name = file.name.split('-').reduce((acc, partialName) => {
+        if (!acc) {
+            return partialName;
+        }
+
+        if (!partialName) {
+            return acc;
+        }
+
+        return `${acc}${partialName[0].toUpperCase()}${partialName.substr(1)}`;
+    }, '');
```

Comment:
This is just converting the string to camelCase, correct? If so, we can just use `camelCase` from `lodash`.